CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- [BREAKING] ProductImageSelection now defaults `stackableVersion` to operator version ([#619]).
```

Comment:
Is this really breaking if the field was mandatory before? 
For all existing and working crds behavior should be unchanged if I am not mistaken?

Reply:
its not necessarily breaking for the end user, but for the clients that use operator-rs (our operators).
From like a Rust code perspective ;)

Reply:
Yeah meaning "you cannot just update the operator-rs version...". Maybe should use CRD_BREAKING and OP_BREAKING or sth like this?